test/e2e/tokenlists.test.ts

```diff
@@ -0,0 +1,67 @@
+import sharp from 'sharp';
+import axios from 'axios';
+import crypto from 'crypto';
+
+const HOST = `http://localhost:${process.env.PORT || 3003}`;
+const cUSDC_TOKEN_ADDRESS_ON_MAIN = '0x39AA39c021dfbaE8faC545936693aC917d5E7563';
+const ERC3770_ADDRESS = 'oeth:0xe0BB0D3DE8c10976511e5030cA403dBf4c25165B';
+const EIP155_ADDRESS = 'eip155:1:0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e';
+
+function getImageFingerprint(input: string) {
+  return crypto
+    .createHash('sha256')
+    .update(input)
+    .digest('hex');
+}
+
+function getImageResponse(identifier: string) {
+  return axios.get(`${HOST}/token/${identifier}?resolver=tokenlists`, {
+    responseType: 'arraybuffer',
+    headers: {
+      'Cache-Control': 'no-cache',
+      Pragma: 'no-cache',
+      Expires: '0'
+    }
+  });
+}
+
+// tokenlists resolver needs a moment on first run
+// there's probably a better way to handle this
+jest.setTimeout(60_000);
+
+describe('tokenlist resolver', () => {
+  it('returns an image for standard address', async () => {
+    const response = await getImageResponse(cUSDC_TOKEN_ADDRESS_ON_MAIN);
+
+    expect(response.status).toBe(200);
+    expect(response.headers['content-type']).toBe('image/webp');
+  });
+
+  it('returns correct image for ERC3770 address', async () => {
+    const response = await getImageResponse(ERC3770_ADDRESS);
+
+    const image = sharp(response.data);
+    const imageBuffer = await image.raw().toBuffer();
+
+    const fingerprint = getImageFingerprint(imageBuffer.toString('hex'));
+    const expectedFingerprint = 'ac601f072065d4d03e6ef906c1dc3074d7ad52b9c715d0db6941ec89bf2073a1';
```

Comment:
As mentioned here https://github.com/stamp-labs/stamp/pull/282#issuecomment-2292623553 this is not the expected image

Reply:
This is not working due to something unrelated to this PR: the shortname `oeth` is not supported:

https://github.com/stamp-labs/stamp/blob/3b50d97093d4f038003cdb8a4abb401e311bc8a0/src/utils.ts#L50-L58